swift/llm/utils/argument.py

```diff
@@ -1196,6 +1196,10 @@ def _init_training_args(self) -> None:
         if 'accelerator_config' in parameters:
             kwargs['accelerator_config'] = {'dispatch_batches': False}
 
+        metric_for_best_model = 'rouge-l' if self.predict_with_generate else 'loss'
+        if hasattr(self, 'rlhf_type') and self.rlhf_type == 'ppo':
+            metric_for_best_model = None
```

Comment:
In the PPO training metrics, there are no metrics that start with "eval", set None here